apps/dashboard/app/views/batch_connect/sessions/connections/_many_tabs.html.erb

```diff
@@ -0,0 +1,14 @@
+<ul class="nav nav-tabs" style=".session-panel .nav.nav-tabs li:only-child { display: none; }">

```

Comment:
I would move `.session-panel .nav.nav-tabs li:only-child { display: none; }` to https://github.com/OSC/ondemand/blob/649bb68fa2d673bbe64b42e6aa83c542cc05c946/apps/dashboard/app/assets/stylesheets/batch_connect/sessions.scss